src/mmw/js/src/draw/views.js

```diff
@@ -41,11 +41,22 @@ function actOnLayer(datum) {
     }
 }
 
-// Takes a GeoJSON shape, or if a key is specified, a dict
-// with a GeoJSON object found at shape[shapeKey]
-function validateShape(shape, shapeKey) {
```

Comment:
Removing this will actually fix a newly reported bug https://github.com/WikiWatershed/model-my-watershed/issues/1166